.github/workflows/bmad-qa-gate-fast.yml

```
@@ -0,0 +1,71 @@
+jobs:
+  qa-gate:
+    # Use self-hosted runner if available, fallback to GitHub-hosted
+    runs-on: ${{ github.repository == 'auldsyababua/bigsirflrts' && github.actor == 'auldsyababua' && 'self-hosted' || 'ubuntu-latest' }}
+    timeout-minutes: 15
+    
+    # Continue on GitHub-hosted if self-hosted is unavailable
+    continue-on-error: ${{ matrix.runner == 'self-hosted' }}
+    
+    steps:
+      - name: Runner Info
+        run: |
```

actionlint (1.7.7)

property "labels" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)